charts/docker-hub-rate-limit-exporter-chart/values.yaml

```diff
@@ -33,29 +60,30 @@ serviceAccount:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name: ""
-
-# Labels to add to each pod.
-podLabels: {}
-    # myPodLabel: labelName
+    imagePullSecrets: []
 
 podAnnotations: {}
 
-podSecurityContext:
-    {}
-    # fsGroup: 2000
-
 securityContext:
-    {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
+  fsGroup: 65534
```

Comment:
I think this would break deployments to Openshift (OCP) clusters since OCP by default disallows to specify an arbitrary userId / groupId for podSecurityContext. 

I even think that it is not possible to "unset" this values for a user when templating or applying the helm chart. Therefore, I think it is necessary to remove this defaults. Although, it would probably improve security for non OCP deployments.